typescript/sdk/src/core/HyperlaneCoreDeployer.ts

```diff
@@ -81,8 +81,9 @@ export class HyperlaneCoreDeployer extends HyperlaneDeployer<
       mailbox.initialize(
         owner,
         defaultIsm,
-        merkleTreeHook.address,
-        merkleTreeHook.address,
+        defaultHook,
+        defaultHook,
+        // merkleTreeHook.address,
```

Comment:
Not sure whether we want to commit this. Probably easiest to not change the mailbox setup but know that it was tested "manually" for both double insertion and zero insertion?